### DISCUSSION TITLE
BERTopic DTM: adjusting for relative frequencies over time

### DISCUSSION THREAD

**Discussion by s0urskittles** [MaartenGr/BERTopic#2059] [Q&A]
I'm looking at changes in topics over time and in my dataset, the actual number of documents increases significantly over time e.g., 2010 has 100 datapoints but 2023 has 500,000. I am aware that you can normalise the frequencies when visualising these changes over time but this just transforms all frequencies to be between 0 to 1 and from my understanding, this does not account for changes in frequency over time. I want to be able to normalise the frequencies so the visualisation can display relative changes in frequencies rather than absolute changes. I can't seem to find any info about this online, does anyone have any advice for this? Thanks in advance!

**Comment by MaartenGr** [comment]
I'm not sure if this is what you want but the `nr_bins` parameter makes sure that each bin has a similar number of documents per bin. Other than that, you can apply any normalization yourself on the resulting dataframe that you get out of `.topics_over_time`.
